nixos/tests/all-tests.nix

```diff
@@ -315,7 +315,7 @@ in
   systemd = handleTest ./systemd.nix {};
   systemd-analyze = handleTest ./systemd-analyze.nix {};
   systemd-binfmt = handleTestOn ["x86_64-linux"] ./systemd-binfmt.nix {};
-  systemd-boot = handleTestOn ["x86_64-linux"] ./systemd-boot.nix {};
+  systemd-boot = handleTestOn ./systemd-boot.nix {};
```

Comment:
Should `handleTestOn` become `handleTest` ?

Reply:
Thanks for catching this!